packages/vx-axis/src/axis/AxisRadial.js

```diff
@@ -0,0 +1,287 @@
+  let format = scale.tickFormat ? scale.tickFormat() : identity;
+  if (tickFormat) format = tickFormat;
+
+  let tickLabelFontSize = 10; // track the max tick label size to compute label offset
+
+  const ticks = values
+    .map((val, index) => {
+      if (hideZero && val === 0) return null;
+
+      const angle = scale(val) - Math.PI / 2;
+      const tickFromPoint = new Point(polarToCartesian({ angle, radius }));
+      const tickToPoint = new Point(polarToCartesian({ angle, radius: radius + tickLength }));
+
+      const tickOrientationProps = radialTickLabelTransform(tickFromPoint, tickToPoint);
+      const tickLabelPropsObj = { ...tickOrientationProps, ...tickLabelProps(val, index) };
+      tickLabelFontSize = Math.max(tickLabelFontSize, tickLabelPropsObj.fontSize || 0);
+
+      return tickLabelPropsObj;
+    })
+    .filter(defined => !!defined);
+
+  const axisLineData = scale.ticks(50);
+  const angle = d => scale(d);
+
+  if (children) {
+    return (
+      <Group className={cx('vx-axis-radial', axisClassName)} top={top} left={left}>
+        {children({
+          angle,
+          axisLineData,
+          numTicks,
+          label,
+          radius,
+          tickLength,
+          tickFormat: format,
+          ticks
+        })}
+      </Group>
+    );
+  }
+
+  return (
+    <Group className={cx('vx-axis-radial', axisClassName)} top={top} left={left}>
+      {values.map((val, index) => {
+        if (hideZero && val === 0) return null;
+
+        const tickAngle = scale(val) - Math.PI / 2; // @TODO why is this off by 45°?
+        const tickFromPoint = new Point(polarToCartesian({ angle: tickAngle, radius }));
+        const tickToPoint = new Point(
+          polarToCartesian({ angle: tickAngle, radius: radius + tickLength })
+        );
+
+        const tickOrientationProps = radialTickLabelTransform(tickFromPoint, tickToPoint);
+        const tickLabelPropsObj = {
+          ...tickOrientationProps,
+          ...tickLabelProps(val, index)
+        };
+        const formattedValue = format(val, index);
+
+        return (
+          <Group
+            key={`vx-tick-${val}-${index}`}
+            className={cx('vx-axis-tick', tickClassName)}
+            transform={tickTransform}
+          >
+            {!hideTicks && <Line from={tickFromPoint} to={tickToPoint} stroke={tickStroke} />}
+            {tickComponent ? (
+              tickComponent({
+                formattedValue,
+                ...tickLabelPropsObj
+              })
+            ) : (
+              <Text {...tickLabelPropsObj}>{formattedValue}</Text>
+            )}
+          </Group>
+        );
+      })}
+
+      {!hideAxisLine && (
+        <LineRadial
+          className={cx('vx-axis-line', axisLineClassName)}
+          data={axisLineData}
+          angle={angle}
+          radius={radius}
+          stroke={stroke}
+          strokeWidth={strokeWidth}
+          strokeDasharray={strokeDasharray}
+          curve={curveCardinal}
```

Comment:
does this always have to be `curveCardinal`? should it be configurable via prop?

Reply:
I think you'd probably want this most of the time, otherwise the axis wouldn't look like an arc.

In thinking about this more though, I changed the implementation of `GridRadial` to use `Arc` instead of `LineRadial`, that could simplify this as well and allow us to drop the `@vx/curve` dep. With `LineRadial` we're sort of doing the arc interpolation ourself but `Arc` should be best at that.